src/django_upgrade/fixers/request_headers.py

```diff
@@ -110,3 +148,11 @@ def rewrite_header_access(tokens: list[Token], i: int, *, raw_header_name: str)
     str_idx = find(tokens, meta_idx, name=STRING)
     header_name = "-".join(x.title() for x in raw_header_name.split("_"))
     replace(tokens, str_idx, src=repr(header_name))
+
+
+def rewrite_in_statement(tokens: list[Token], i: int, *, raw_header_name: str) -> None:
+    meta_idx = find(tokens, i, name=NAME, src="META")
+    replace(tokens, meta_idx, src="headers")
+    str_idx = reverse_find(tokens, meta_idx, name=STRING)
+    header_name = "-".join(x.title() for x in raw_header_name.split("_"))
```

Comment:
This is duplicated from the other rewrite function - extract it into a separate function called `make_header_name` ? Or maybe `get_http_header_name` can be rewritten to do this, kinda wasteful to process the string twice